packages/action-password-update/src/components/PasswordUpdateDialog/PasswordInput.js

```diff
@@ -10,6 +10,7 @@ const PasswordInput = props => (
       value={props.value}
       onChange={e => props.onChange(e.target.value)}
       readOnly={props.readOnly === true}
+      disabled={props.readOnly === true}
```

Comment:
It makes sense to use attribute disabled only. I have tested all possible attribute combinations with https://jsfiddle.net/niklaushug/yd1LL9kf/ for newest browser versions: FF, CH, SA, OP as well as IEE, IE11, IE10 and IE9. Apart of varying default stylings for attribute readonly the behaviour is for all browsers the same.

attribute readonly: protects value, adds some default styling
attribute disabled: protects value, prevent focus by tab key, does not submit value with post request